doc/compiler/jitserver/Usage.md

```diff
@@ -20,143 +20,243 @@ OpenJDK Assembly Exception [2].
 SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-# Usage
+# Getting started with JITServer
 
-This document describes how to use JITServer technology with any Java application and some basic options, such as logging and encryption.
+This document explains how to use JITServer technology with any Java application and describes some basic options, such as network configuration, logging and encryption.
+
+1. [Running applications with JITServer](#running-applications-with-jitserver)
+2. [Network Configuration](#network-configuration)
+3. [JITServer-specific options](#jitserver-specific-options)
+4. [Logging](#logging)
 
 ## Running applications with JITServer
 
-JITServer adds two additional *personas* to the JVM: client mode and server mode. To start the JVM in server mode, run the `jitserver` launcher present under `bin` directory (along side `java` launcher). To start the JVM in client mode, there is a new command line option: `-XX:+UseJITServer` (under bin run `./java -XX:+UseJITServer <application name>`)
+JITServer adds two additional *personas* to the JVM: **client** mode and **server** mode.
+**Client** is the Java application you want to run, and **server** performs JIT compilations for the **client**.
+
+In **server** mode, the JVM will halt after startup and begin listening for compilation requests from clients. To start the **server**, run the `jitserver` launcher present under `bin` directory (alongside `java` launcher). No Java application is given to the `jitserver` launcher on the command line. Any option accepted by JVM can also be passed to the `jitserver` launcher. One **server** is capable of serving multiple clients.
 
-In server mode, the JVM will halt after startup and begin listening for compilation requests from clients. No Java application is given to the `jitserver` launcher on the command line. Any option accepted by JVM can also be passed to the `jitserver` launcher.
+Begin by launching the **server**:
 
 ```
-$ jitserver
+$ jitserver -XX:+JITServerLogConnections
 JITServer is currently a technology preview. Its use is not yet supported
 
 JITServer is ready to accept incoming requests
 
 ```
 
-Clients can then be started. For this example, we assume the client and server are running on the same machine. The server needs to continue running, so you will need to start clients in a new terminal.
+> **Note**: we passed an option `-XX:+JITServerLogConnections` to the server to enable logging of client connection events. Refer to [this](#connection-logging) section for more information.
+
+With the server running, clients can now be started. To start the JVM in **client** mode, provide a command line option: `-XX:+UseJITServer` (under `bin` run `./java -XX:+UseJITServer <application name>`)
```

Comment:
To start the JVM in **client** mode, provide the `-XX:+UseJITServer` command line option to the java launcher. In the following example we assume the **client** and the **server** are run on the same machine. Since the **server** needs to continue running, we need to start the client in a new terminal.
...